package.json

```diff
@@ -78,7 +78,6 @@
     "ember-cli-import-polyfill": "^0.2.0",
     "ember-cli-is-package-missing": "^1.0.0",
     "ember-cookies": "^0.0.10",
-    "ember-getowner-polyfill": "^1.0.1",
```

Comment:
Yes, we'll want to maintain the current compatibility unless not possible anymore.